x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -866,3 +876,32 @@ export function resolveMaxTimeInterval(timeIntervals: string[]): number | undefi
 
   return Number.isFinite(result) ? result : undefined;
 }
+
+export const getFiltersForDSLQuery = (
```

Comment:
Moving this function out of anomalies table.
I'm not 100% on having it in `common/util/job_utils.ts` as this file is quite big. It could be broken up into multiple files inside `job_utils` folder.